.github/workflows/ci.yml

```diff
@@ -1,6 +1,11 @@
 name: CI
 
-on: [push]
+on:
+  push:
+  schedule:
+  # Run every fifth minute.
```

Comment:
Is there a rate limit to how much we can run GH actions, even on public repos ?

Reply:
[Doesn't look like it](https://docs.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/managing-your-spending-limit-for-github-actions), although 5 minutes is [the shortest available interval](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule). If there is, hopefully six days will be enough to hit it, so we can fix it before next month :)